### DISCUSSION TITLE
Start the NSPanel to the thermostat page in case of HA is offline

### DISCUSSION THREAD

**Discussion by lookasek** [Blackymas/NSPanel_HA_Blueprint#1431] [Q&A]
Hi,
I would like to know if is it possible to start NSPanel to the thermostat page when HA is offline?

I would like to be able to control the embedded thermostat after restarting the power supply in case of the HA is offline.

**Comment by edwardtfn** [comment]
Yes, it should be possible when using the add-on climate.
If HA gets unavailable, you can keep controlling it normally from your panel.
In a case of a power failure, your panel will also fail. When it gets back, it will wait for Home Assistant for a few seconds, but then will show the Home page in the offline mode, from where you can also open the climate page and control the thermostat even while HA is not available.

**Reply by lookasek** [reply]
I have just done a test (v4.1.4), but after restarting the power, the device still displays the Initializing page.

**Reply by edwardtfn** [reply]
mmm
It looks like a bug.
I will take a look.

**Reply by edwardtfn** [reply]
I've created #1432 for this.
